dtfs-central-api/src/services/utilisation-report-data-validator/utilisation-report-cell-validators/validate-csv-data/rows/index.ts

```diff
@@ -28,9 +28,8 @@ export const validateRows = (csvData: UtilisationReportCsvRowData[]): Utilisatio
     const ukefFacilityId = row['ukef facility id']?.value;
     const baseCurrencyValue = row['base currency']?.value;
     const facilityUtilisationValue = row['facility utilisation']?.value;
-    const exporterName = row['bank facility reference']?.value;
 
-    if (!baseCurrencyValue || !facilityUtilisationValue || !ukefFacilityId || !exporterName) {
+    if (!baseCurrencyValue || !facilityUtilisationValue || !ukefFacilityId) {
```

Comment:
Is there a unit test for this? Worrying if it's passing without any changes

Reply:
there was no unit test for this, so going to add some as part of the https://ukef-dtfs.atlassian.net/browse/FN-3643 as well as using the const for pulling out the values instead of hard coded strings!